src/boards/include/boards/wiznet_w5100s_evb_pico2.h

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// -----------------------------------------------------
+// NOTE: THIS HEADER IS ALSO INCLUDED BY ASSEMBLER SO
+//       SHOULD ONLY CONSIST OF PREPROCESSOR DIRECTIVES
+// -----------------------------------------------------
+
+// pico_cmake_set PICO_PLATFORM=rp2350
```

Comment:
The syntax of these cmake-specific lines have changed - see https://github.com/raspberrypi/pico-sdk/pull/2397 and https://github.com/raspberrypi/pico-sdk/blob/develop/src/boards/include/boards/wiznet_w5100s_evb_pico.h